### ISSUE TITLE
Source zip file on SourceForge is not correct again...

### ISSUE THREAD

**Issue by ChenYuChih** [CoolProp/CoolProp#1292]
Dear development team,

I  download the latest(v6.1.0) version source zip file at "https://sourceforge.net/projects/coolprop/files/CoolProp/6.1.0/source/" and found that the content is not correct. The ".version" file says its 5.1.3 and the total file size is not right, either.(Much smaller than v6.0.0) Maybe there's something wrong during updating. Please check it. 

Thank you very much for your help!

Best Regards,
Chen, Yu-Chih

**Comment by JonWel** [comment]
I confirm the issue (I also get 5.1.3, with no cubic backend inside)

**Comment by jowr** [comment]
I think I fixed it... Can anyone confirm that it works now.

**Comment by JonWel** [comment]
This looks better: `6.1.1dev` and the cubic's file are present.

**Comment by jowr** [comment]
Are you sure? Should be 6.1.0 - please reopen if not 6.1.0.
```Bash
cd tmp
git clone https://github.com/CoolProp/CoolProp.git CoolProp.git
cd CoolProp.git
git checkout -b v6.1.0 tags/v6.1.0
git submodule init
git submodule update
cd ..
rsync -a --delete CoolProp.git/ CoolProp.sources
cd CoolProp.sources
python "dev/generate_headers.py"
find . -iwholename "*/.git*" -exec rm -rf {} \;
cd ..
rm -f CoolProp_sources.zip
zip -rq CoolProp_sources.zip CoolProp.sources
rsync -a --no-perms -z --stats CoolProp_sources.zip frs.sf.net-jorritw:/home/frs/project/coolprop/CoolProp/6.1.0/source/
cd ..
```

**Comment by JonWel** [comment]
@jowr  That's what I had in the file `.version` when downloading soon after your comment.
I downloaded again and it is now the correct `6.1.0`.
So this looks solved to me.

**Comment by ibell** [comment]
Argh.  I think I know why, but it's going to be a while until we can get it
updated.  Do you have to use the sources?  Can you not just use the git
checkout?

**Comment by jowr** [comment]
@ibell, what I the problem? I am quite sure that everything is fine and @JonWel just confirmed that...

**Comment by ChenYuChih** [comment]
Thanks for your help! It seems quite fine now.

@ibell, I prefer downloading the zip file from sourceforge because I cannot get the complete project dependencies from github "Clone or download" button(Master branch). For example, the externals folder is empty and some headers("all_cubics_JSON.h", "all_fluids_JSON.h", "all_incompressibles_JSON.h" and so on) are missed also. Did I do anything wrong?

Best Regards,
Chen, Yu-Chih

**Comment by ibell** [comment]
@ChenYuChih You are not doing anything wrong, but you should become familiar with how to use git, it will make your life much simpler, and then you can grab intermediate copies of the code and not have to wait for a release to come out

**Comment by ibell** [comment]
Yes I think we are fine for now unless something else comes up

**Comment by ibell** [comment]
Closed by a9d420caaac68be528591327552e7851a548cb94
